src/frontend/initApp.js

```diff
@@ -21,12 +24,15 @@ export default async function initApp() {
 
     await store.dispatch(init());
 
+    const history = createHistory();
+    syncReduxAndRouter(history, store);
+
     const component = (
       <Provider store={store}>
         <ReduxIntlContainer>
-          <ReduxRouter
-            RoutingContext={RelayRoutingContext}
-          />
+          <Router RoutingContext={RelayRoutingContext} >
```

Comment:
Please, pass `routes` via props, because we're using alternative configuration.
https://github.com/rackt/react-router/blob/master/docs/guides/basics/RouteConfiguration.md#alternate-configuration